versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.14.0"
+  required_version = ">= 1.3"
```

Comment:
Why does it require 1.3+?

Reply:
There is no strict requirement for 1.3+ in the docs, but I've noticed a tendency across the several other repos in the org to update `hashicorp/aws` to 5+ along with TF 1.3+. Do you want me to undo this constraint?

Reply:
No, I think it makes sense. We should bump major anyway because of the provider version bump I'm thinking, so bumping the required TF version makes sense as well?

Reply:
Ah just seeing your comment @Benbentwo -- Thoughts on rationale above?